Header header
uint32 form
sensor_msgs/Image image
geometry_msgs/PoseStamped pose
float64 confidence
